cogs/study_groups.py

```diff
@@ -134,6 +134,43 @@ async def _group(self, ctx: Context):
         member: Union[Member, User] = ctx.author
         logger.info(f'User="{member.name}#{member.discriminator}({member.id})", Command="{ctx.message.content}"')
 
+    @_group.command(pass_context=True,
+                    name="create",
+                    brief="Creates a Study group.",
+                    help="The name must contain the tag and the semester number.")
+    async def group_create(self, ctx: Context, name: str, semester: int = 1, color: int = 0):
```

Comment:
make color a needed parameter with no default, force the User to give one.

Reply:
Okay is required now